views/tasks/view.php

```diff
@@ -12,7 +12,9 @@
     <div class="left-column">
         <div class="head-wrapper">
             <h3 class="head-main"><?=$task->title?></h3>
-            <p class="price price--big"><?=$task->budget?>&nbsp;₽</p>
+            <p class="price price--big">
+                <?=$task->budget?><?=$task->budget ? '&nbsp;₽' : ''?>
```

Comment:
тут для дописывания валюты оптимально использовать форматтер фреймворка:
```suggestion
                <?= Yii::$app->formatter->asCurrency($task->budget) ?>
```

Reply:
Если сделать так, то а) появляются копейки после запятой, б) в заданиях, где стоимость не указана, появляется "0 ₽". По сути с использованием форматтера можно сделать вот так:
`<?=$task->budget ? Yii::$app->formatter->asCurrency($task->budget, 'RUB',[NumberFormatter::FRACTION_DIGITS => 0]) : ''?>`
Но так как будто хуже читается. Что лучше сделать в таком случае? Оставить мой вариант, или использовать форматтер?